crates/fj-kernel/src/algorithms/approx/sketch.rs

```diff
@@ -12,9 +12,9 @@ impl Approx for &Sketch {
 
     fn approx_with_cache(
         self,
-        tolerance: impl Into<Tolerance>,
-        cache: &mut Self::Cache,
+        _tolerance: impl Into<Tolerance>,
+        _cache: &mut Self::Cache,
     ) -> Self::Approximation {
-        self.faces().approx_with_cache(tolerance, cache)
+        todo!()
```

Comment:
Left this as a `todo!()` for now as I'm not aware that we need to approximate `Sketch`es directly _yet_, we always first turn them into faces/shells and then approximate?

Reply:
Yeah, fair enough. We can fill this in once required :+1: